tests/caasp/stack_conformance.pm

```diff
@@ -25,6 +25,12 @@ sub run {
 
     # https://github.com/cncf/k8s-conformance/blob/master/instructions.md
     my $sb_yaml = 'https://raw.githubusercontent.com/cncf/k8s-conformance/master/sonobuoy-conformance.yaml';
+
+    # CaaSP 2.0 has Kubernetes 1.8.9 which doesn't work with v1.10.0 testsuite
+    if (is_caasp('qam')) {
+        my $sb_yaml = 'https://raw.githubusercontent.com/cncf/k8s-conformance/6179d790e6bfc799afef5058ce50a2f314983fa2/sonobuoy-conformance.yaml';
```

Comment:
you are redefining existing variable, remove `my`

Reply:
Also your local run does not run conformance test - don't you need to import `version_utils 'is_caasp'` ?